openapi-specs/schema.json

```diff
@@ -942,8 +942,20 @@
     },
     "BackupConfig": {
       "description": "Backup custom configuration",
-      "type": "object",      
+      "type": "object",
       "properties": {
+        "Endpoint": {
+          "type": "string",
+          "description": "S3 endpoint, e.g. s3.amazonaws.com"
```

Comment:
here also we could put a more generic description like "name of the endpoint to overrirde e.g. s3.amazonaws.com"

Reply:
and can we use this more generic description also in the subsequent definitions